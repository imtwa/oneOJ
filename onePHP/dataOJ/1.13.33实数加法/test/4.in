13858448386.8186337137487122615752736437736442877811787177226812311465484136618176242326662
52648116821.3654517746644221617834176765228215381432163554483752618262652232711258478668438
