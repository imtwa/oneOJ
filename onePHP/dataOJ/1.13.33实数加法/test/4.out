66506565208.18408548841313442335869132029646582592439507317105649297281363693294347209951